src/docs/patterns/forms.mdx

```diff
@@ -82,3 +95,78 @@ In some specific situations, you may need to defer verification or perform it fo
         '2: Use a circular progress indicator when waiting for responses from the server.',
     ]}
 />
+
+## Submit the Form
+
+Usually, a form will have a submit/next/finished button at the end. This button is typically placed at the bottom right corner of the form. If you are designing for a right-to-left language, this button should be at the bottom left.
+
+<ImageGrid
+    images={[SumbitButtonLocationImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        '1: On desktops, the button is placed in the bottom corner of the form.',
+        '2: On mobile devices, the button is occluded by the native keyboard.',
+    ]}
+/>
+<ImageGrid
+    images={[NextButtonUsageImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        'On mobile, if there are more than one input fields on the screen, the keyboard button should say "next".',
+        '1: The user entered a valid input. Clicking "next" will autofocus the user to the next text field.',
+        '2: The user entered a invalid input. Clicking "next" will display the error message but will not autofocus to the next text field.'
+    ]}
+/>
+
+### Disable the Submit Button
+
+When there are not a lot of text fields, the submit button can be disabled to indicate that there are some errors in the user input.
+
+<ImageGrid
+    images={[SubmitButtonOnChangeImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'If you have only one input field, and this field is verified on change, disable the submit button until the user entered the desired input.'
+    }
+/>
+<ImageGrid
+    images={[SubmitButtonAutoSubmitImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={
+        'For the verification code screen, you can auto-submit the code once the user input reaches the desired length (2). After the verification code is recognized (3), the screen pauses for a short moment before auto-transit to the next page. The verify button is preserved for consistency.'
```

Comment:
```suggestion
        'For the verification code screen, you can auto-submit the information once the user input reaches the required length (#2). After the verification code is recognized (#3), the screen pauses for a short moment before automatically moving to the next screen. The verify button is preserved for consistency.'
```
"The verify button is preserved for consistency" - What does this mean? From the description it sounds like the verification button is not used at all, so why even have one?

This workflow seems like a special case. Does it warrant being called out at all? And if so, should it be more prominent than just an image caption?

Reply:
For the consistency -- if a submit button was there for all the other patterns except for this one, it might feel strange. Although in WAS, instead of "submit", it was a wide "resend confirmation email" button instead. Maybe I should change it -- want to get some big reactions from Ian.

I in fact wanted to put it to login workflow, but we have more contents on the form validation side at the moment, it seems better to have it here